demo-notebooks/guided-demos/widget_notebook_example.ipynb

```diff
@@ -0,0 +1,119 @@
+{
+ "cells": [
```

Comment:
nit: Can we name this `3_widget_example.ipynb` to follow suit with the naming scheme of the others